documentation/develop/build-an-accessible-extension.md

```diff
@@ -48,7 +48,7 @@ There is more information on design and accessibility in the [Photon Design Syst
 <tr>
 <td markdown="1">
 
-[Keyboard shortcuts (commands)](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/commands)
+[Keyboard shortcuts (commands)](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/commands)
```

Comment:
All the links should not have `/en-US` which is the locale part in urls. For example here URL should be `https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/commands`

Reply:
okay!